### ISSUE TITLE
cmark-gfm build to .travis.yml

### ISSUE THREAD

**Issue by dvorka** [dvorka/mindforger#942]
Don't remove discount code completely, but comment it so that it can be activated later.

**Comment by dvorka** [comment]
Unit test `.pro` seems to be broken:
```
g++ -m64 -Wl,-O1 -o mindforger-lib-unit-tests datetime_test.o string_utils_test.o repository_indexer_test.o markdown_test.o fts_test.o memory_test.o mind_test.o note_test.o mindforger_lib_unit_tests.o outline_test.o test_gear.o configuration_test.o markdown_benchmark.o html_benchmark.o html_test.o nlp_test.o trie_benchmark.o ai_benchmark.o file_utils_test.o trie_test.o autolinking_test.o   -L/home/travis/build/dvorka/mindforger/lib/test/src/../../../lib -lmindforger -L/home/travis/build/dvorka/mindforger/lib/test/src/../../../deps/discount -ldiscount -lz -lgtest -lpthread 
/home/travis/build/dvorka/mindforger/lib/test/src/../../../lib/libmindforger.a(cmark_gfm_markdown_transcoder.o): In function `m8r::CmarkGfmMarkdownTranscoder::to(m8r::RepresentationType, std::string const*, std::string*)':
cmark_gfm_markdown_transcoder.cpp:(.text+0x91): undefined reference to `cmark_gfm_core_extensions_ensure_registered'
cmark_gfm_markdown_transcoder.cpp:(.text+0x96): undefined reference to `cmark_get_default_mem_allocator'
cmark_gfm_markdown_transcoder.cpp:(.text+0xa1): undefined reference to `cmark_list_syntax_extensions'
cmark_gfm_markdown_transcoder.cpp:(.text+0xae): undefined reference to `cmark_parser_new'
cmark_gfm_markdown_transcoder.cpp:(.text+0xc8): undefined reference to `cmark_parser_attach_syntax_extension'
cmark_gfm_markdown_transcoder.cpp:(.text+0xe0): undefined reference to `cmark_parser_feed'
cmark_gfm_markdown_transcoder.cpp:(.text+0xe8): undefined reference to `cmark_parser_finish'
cmark_gfm_markdown_transcoder.cpp:(.text+0x107): undefined reference to `cmark_render_html_with_mem'
cmark_gfm_markdown_transcoder.cpp:(.text+0x135): undefined reference to `cmark_node_free'
cmark_gfm_markdown_transcoder.cpp:(.text+0x140): undefined reference to `cmark_llist_free'
cmark_gfm_markdown_transcoder.cpp:(.text+0x148): undefined reference to `cmark_parser_free'
collect2: error: ld returned 1 exit status
make[1]: *** [mindforger-lib-unit-tests] Error 1
make[1]: Leaving directory `/home/travis/build/dvorka/mindforger/lib/test/src'
make: *** [sub-src-make_first] Error 2
The command "make" exited with 2.
0.01s$ cd src && export M8R_GIT_PATH="${M8RHOME}" && ./mindforger-lib-unit-tests > /tmp/mindforger-unit-tests.log 2>&1
The command "cd src && export M8R_GIT_PATH="${M8RHOME}" && ./mindforger-lib-unit-tests > /tmp/mindforger-unit-tests.log 2>&1" exited with 127.
```

**Comment by dvorka** [comment]
Fixed by 4e36f97bac34b6d70473497a48d9f998af0668e2
